<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugTools</class>
 <widget class="QDialog" name="DebugTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug Tools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Packet Injector</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>History:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="clearBtn">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="historyListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Packet:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="packetLineEdit">
          <property name="placeholderText">
           <string>{P0|Roll|255,0,0|25}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendPacketBtn">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>packetLineEdit</tabstop>
  <tabstop>sendPacketBtn</tabstop>
  <tabstop>historyListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
